.github/workflows/main.yml

```diff
@@ -0,0 +1,138 @@
+name: main
+'on':
+  - workflow_dispatch
```

Comment:
Right now it will be manual trigger and later it will be a scheduled cron job I assume.

Reply:
yes